com.github.Inochi2D.Inochi-Creator.yml

```diff
@@ -0,0 +1,164 @@
+id: com.github.Inochi2D.Inochi-Creator
+runtime: org.kde.Platform
+runtime-version: "5.15-21.08"
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.ldc
+  - org.freedesktop.Sdk.Extension.dmd
+command: "inochi-creator"
+finish-args:
+    - "--device=all" #OpenGL rendering, webcams (may be required? not sure)
+    - "--socket=x11"
+    - "--share=ipc"
+    - "--socket=wayland"
+    - "--filesystem=host" #This is a dev tool, it should probably have as much access to the host as it can.
+    - "--filesystem=home" #So that it can access the project files, should they be located in the home folder

```

Comment:
1. this is broad
2. host include home

Reply:
This is addressed by the fact that the app does not yet utilize portals, so I give it access to everything to cover where the project files could be located. As for having both of them set, there's not really a specific reason.